### ISSUE TITLE
Handle the case where another AT has hooked touch in a user friendly way

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3094]
**Reported by LJWatson on 2013-03-19 11:17**
Using a Microsoft Surface Pro. Touch gestures do notseem to work.

NVDA does not respond when placing a finger on the screen, when moving a finger across the screen, or when using gestures such as tap, double tap etc.

NVDA does not respond with any announcement to idicate the gesture has been successful, and subsequent ivestigation with the keyboard suggests that the gesture was not recognised itself.

For example: On the start screen with focus on the Mail app. NVDA does not announce the app when placing a finger on the screen. NVDA does not respond when either a left or right flick is used, and subsequent investigation with the keyboard indicates that focus has not moved to the next app either.

**Comment by Adriani90** [comment]
@LJWatson is this still an issue for you?
@jcsteh, @michaelDCurran in case this is still reproducible maybe there should be a global touch gesture for exiting Narator and a global touch gesture for starting NVDA.

**Comment by LJWatson** [comment]
@Adriani90 I'm sorry, I don't know. I no longer have my Surface device.

**Comment by Adriani90** [comment]
Closing as abandoned. we don't have any tests so far, noone provided further updates. If there is still an issue, please consider opening a new one filling the bug template, or comment on this one with updated test results and we can reopen.
